reactor-core/src/main/java/reactor/core/publisher/Mono.java

```diff
@@ -2010,7 +2010,9 @@ public T block(Duration timeout) {
 	 * @param afterSuccessOrError the callback to call after {@link Subscriber#onNext}, {@link Subscriber#onComplete} without preceding {@link Subscriber#onNext} or {@link Subscriber#onError}
 	 *
 	 * @return a new {@link Mono}
+	 * @deprecated prefer using {@link #doAfterTerminate(Runnable)}. will be removed in 3.4.0
```

Comment:
The previous one had an advantage of the access to `Throwable`.
Perhaps we should explain how to migrate old code that was accessing it?

Reply:
suggested a change :)